### PR TITLE
chore: updated contributing section in README.md

### PR DESCRIPTION
**Pull request by christyjacob4**
# Title
Improved the contributing section in the readme

## Description of what's changing
Added a few more instructions for new contributors 
Corrected the numbering in the README 

## What else might be impacted?
None 

## Which issue does this PR relate to?
#30 

## Checklist

[x] Tests are written and maintain or improve code coverage
[x] I've tested this in my application using `yarn link` (if applicable)
[x] You consent to and are confident this change can be released with no regression

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,13 +47,14 @@ You can find a demo of the project [here](https://github.com/reubenae/user-data-
...
-1. Make your changes.
-1. Make sure it builds `yarn build` or `npm build`
-1. Run the tests (you added tests, right?) with `npm test` or `yarn tests`
-1. Test your changes in your consuming code: `yarn link` [Yarn link documentation](https://classic.yarnpkg.com/en/docs/cli/link) or `npm link` [npm link documentation](https://docs.npmjs.com/cli/link)
-1. Ensure the code coverage is the same or higher than before your changes.
-1. Create a PR to the `master` branch
+2. Install dependencies locally `yarn` or `npm install`
+3. Make your changes.
+4. Make sure it builds `yarn build` or `npm build`
+5. Run the tests (you added tests, right?) with `npm test` or `yarn tests`
```
Comment (reubenae) [inline README.md:53]: Please revert this change. Numbered lists in markdown are supposed to be all 1's - when the markdown is processed they get the correct numbers added. It means you can insert an extra step earlier on without having to modify all the subsequent steps.

Rendered view in master: 
image

File: .env [hunk 2]
```diff
@@ -0,0 +1,4 @@
+NPM_TOKEN=95ba9f82-8ec6-4a44-8209-85e4360cd194
+GH_TOKEN=0a5b546be7a5d248950fd41cf495c31b664fdd3f
```
Comment (reubenae) [inline .env:2]: I don't think you meant to commit this

Comment (christyjacob4) [reply .env:2]: Yea I had added it to my .gitignore 
escaped somehow

**Comment by skodamarthi** [comment]
@christyjacob4 As part of this PR, can you please also do the change to remove additional links by adding links for documentation directly under yarn link and npm link directly. And change it to "Test your changes in your consuming code or using our demo project by running yarn link or npm link."

**Comment by christyjacob4** [comment]
Sure @skodamarthi

**Review by reubenae — APPROVED** [review]

**Comment by christyjacob4** [comment]
Done @skodamarthi

File: .gitignore [hunk 3]
```diff
@@ -1,4 +1,5 @@
 node_modules
 dist
 yarn-error.log
-coverage
\ No newline at end of file
+coverage
+.env
```
Comment (skodamarthi) [inline .gitignore:5]: Did you intend to commit this? Can this be removed as it is not relevant to this PR?

File: README.md [hunk 4]
```diff
@@ -46,12 +46,14 @@ You can find a demo of the project [here](https://github.com/reubenae/user-data-
...
 
+We do not allow contributors to claim issues. If you find something interesting you can contribute to the repo, feel free to raise a PR. We do not require you to let us know in advance. 
+
 1. Fork the repo
 1. Install dependencies locally `yarn` or `npm install`
 1. Make your changes.
 1. Make sure it builds `yarn build` or `npm build`
 1. Run the tests (you added tests, right?) with `npm test` or `yarn tests`
-1. Test your changes in your consuming code: `yarn link` [Yarn link documentation](https://classic.yarnpkg.com/en/docs/cli/link) or `npm link` [npm link documentation](https://docs.npmjs.com/cli/link)
+1. Test your changes in your consuming code: [`yarn link`](https://classic.yarnpkg.com/en/docs/cli/link) or [`npm link`](https://docs.npmjs.com/cli/link)
```
Comment (skodamarthi) [inline README.md:56]: Please change the text to :
Test your changes in your consuming code or using our demo project:

Comment (skodamarthi) [inline README.md:56]:
```suggestion
1. Test your changes in your consuming code or using our demo project: [`yarn link`](https://classic.yarnpkg.com/en/docs/cli/link) or [`npm link`](https://docs.npmjs.com/cli/link)
```

File: .gitignore (hunk 3 above)
Comment (christyjacob4) [reply .gitignore:5]: Yes it's intentional. Would come in handy when we setup `auto`

File: README.md (hunk 4 above)
Comment (christyjacob4) [reply README.md]: Done

**Review by skodamarthi — APPROVED** [review]

**Comment by christyjacob4** [comment]
@skodamarthi There was an update from digital ocean, the hacktoberfest label is mandatory for contributions to be considered. 
https://hacktoberfest.digitalocean.com/hacktoberfest-update

Could you please do the needful

**Comment by reubenae** [comment]
@christyjacob4 no need:

> Your pull requests will count toward your participation if they are in a repository with the hacktoberfest topic and once they have been merged, approved by a maintainer or labelled as hacktoberfest-accepted.

We have the hacktoberfest topic on this repo, so your PR should be appearing anyway

**Pull request merged**